src/frontend/src/lib/services/address.services.ts

```diff
@@ -143,11 +146,17 @@ export const loadIdbAddress = async (): Promise<{ success: boolean }> => {
 	return { success: true };
 };
 
-export const certifyAddress = async (
-	address: string
-): Promise<{ success: boolean; err?: string }> => {
-	const tokenId = ETHEREUM_TOKEN_ID;
-
+const certifyAddress = async <T extends address>({
```

Comment:
Not sure why `extends address` if this function is generic? Eth address does not extends adress or this is just for BTC?

Reply:
no, you are right, it can be without extending, since it is generic